src/app/(sidebar)/transaction/submit/page.tsx

```diff
@@ -220,6 +225,36 @@ export default function SubmitTransaction() {
           variant="success"
           title="Transaction submitted!"
           subtitle={`Transaction succeeded with ${submitRpcResponse.operationCount} operation(s)`}
+          note={<></>}
+          footerLeftEl={
+            IS_BLOCK_EXPLORER_ENABLED ? (
+              <>
+                <Button
+                  size="md"
+                  variant="tertiary"
+                  onClick={() => {
+                    openUrl(
+                      `https://stellar.expert/explorer/${network.id}/tx/${submitRpcResponse.hash}`,
```

Comment:
It looks like mainnet on StellarExpert is `public` ([example](https://stellar.expert/explorer/public/tx/876f9b8a791ef20e9cd4ef9e03d3082f51e1f2db1661b70cb0885ddf946fa771)).

Reply:
thank you! updated it and tested on mainnet